### PR TITLE
Update troubleshooting re: webhook-service error

### PR DESCRIPTION
**Pull request by tpmccallum**

### CODE REVIEW

File: content/en/docs/spin-operator/support/troubleshooting.md [hunk 1]
```diff
@@ -8,6 +8,56 @@ weight: 100
...
+get pods -n spin-operator
+NAME                                                READY   STATUS              RESTARTS   AGE
+spin-operator-controller-manager-5bdcdf577f-htshb   0/2     ContainerCreating   0          26m
+```
+
+If the above result (ready 0/2) is returned, then use the name from the above result to `kubectl describe pod` of the spin-operator:
+
+```console
+kubectl describe pod spin-operator-controller-manager-5bdcdf577f-htshb -n spin-operator
+```
```
Comment (rajatjindal) [inline content/en/docs/spin-operator/support/troubleshooting.md:31]: we should probably show the error here: "cert not ready" error

File: content/en/docs/spin-operator/support/troubleshooting.md [hunk 2]
```diff
@@ -8,6 +8,56 @@ weight: 100
...
+
+In addition, we can check the certificate (the desired output is as follows):
+
```
Comment (rajatjindal) [inline content/en/docs/spin-operator/support/troubleshooting.md:34]: maybe add info like: "this certificate is required by spin-operator to serve webhooks and missing certificate could be one of the reason why spin-operator" is failing to start

File: content/en/docs/spin-operator/support/troubleshooting.md (hunk 1 above)
Comment (tpmccallum) [reply content/en/docs/spin-operator/support/troubleshooting.md:31]: This has been updated now, @rajatjindal - thank you!

File: content/en/docs/spin-operator/support/troubleshooting.md (hunk 2 above)
Comment (tpmccallum) [reply content/en/docs/spin-operator/support/troubleshooting.md:36]: This has been updated now, @rajatjindal - thank you!

**Review by vdice — APPROVED** [review]

**Pull request merged**